lib/prompts/checkbox.js

```diff
@@ -86,7 +90,8 @@ Prompt.prototype.render = function (error) {
   var bottomContent = '';
 
   if (this.firstRender) {
-    message += '(Press <space> to select)';
+    message += '(Press <space> to select, <a> to select all,\n<n> to select none,';
```

Comment:
Got it. I'm wrapping up the changes now but wanted to confirm before committing.

You'd like the 'a' key to 'select all' if every choice isn't selected, but to also 'select none' if all the choices are already selected?

Let me know if this is mistaken from what you were expecting.